third_party/blink/renderer/platform/wtf/hash_table.h

```diff
@@ -1103,6 +1129,55 @@ HashTable<Key, Value, Extractor, HashFunctions, Traits, KeyTraits, Allocator>::
       const_cast<const HashTable*>(this)->Lookup<HashTranslator>(key));
 }
 
+template <typename Key,
+          typename Value,
+          typename Extractor,
+          typename HashFunctions,
+          typename Traits,
+          typename KeyTraits,
+          typename Allocator>
+template <typename HashTranslator, typename T>
+inline ssize_t
+HashTable<Key, Value, Extractor, HashFunctions, Traits, KeyTraits, Allocator>::
+    LookupIdx(const T& key) const {
+  DCHECK(!AccessForbidden());
```

Comment:
DCHECK compiles to no-op in our builds, so you may want to use something better for this if you want it to be actually checked.

Reply:
During rehashing, we build a new hash table, and then steal its guts, replacing the old table.

There's a comment in there that worries me, about how this works with the concurrent collector. I need to go find the code that scans the table, and make sure it only scans the table_ member before this can land.

Reply:
Please also see my other comment regarding `DCHECK` [here](https://github.com/replayio/chromium/pull/616#discussion_r1182622602)